mets_adapter/populate_mets/populate_mets.py

```diff
@@ -0,0 +1,76 @@
+#!/bin/env python3
+
+from sys import maxsize
+import click
+import boto3
+from wellcome_aws_utils.sns_utils import publish_sns_message
+
+
+def aws_resource(name, role_arn):
+    role = boto3.client("sts").assume_role(
+        RoleArn=role_arn, RoleSessionName="AssumeRoleSession1"
+    )
+    credentials = role["Credentials"]
+    return boto3.resource(
+        name,
+        aws_access_key_id=credentials["AccessKeyId"],
+        aws_secret_access_key=credentials["SecretAccessKey"],
+        aws_session_token=credentials["SessionToken"],
+    ).meta.client
+
+
+class StorageManifestScanner:
+
+    role = "arn:aws:iam::975596993436:role/storage-read_only"
+    vhs = "vhs-storage-manifests"
+
+    def __init__(self):
+        self.dynamodb = aws_resource("dynamodb", self.role)
+
+    @property
+    def paginator(self):
+        return self.dynamodb.get_paginator("scan")
+
+    def scan(self):
+        for page in self.paginator.paginate(TableName=self.vhs):
+            for item in page["Items"]:
+                space, id = item["id"].split(":")
+                yield (space, id)
+
+
+class MessagePublisher:
+
+    role = "arn:aws:iam::760097843905:role/platform-dev"
```

Comment:
I can't assume this role for some reason when running locally, anyone got an idea why (`read_only` roles appear to be fine) :thinking:?

```
botocore.exceptions.ClientError:
An error occurred (AccessDenied) when calling the AssumeRole operation: User:
arn:aws:sts::760097843905:assumed-role/platform-superdev/Ni.Ward@wellcome.ac.uk
is not authorized to perform: sts:AssumeRole on resource:
arn:aws:iam::760097843905:role/platform-dev
```

Reply:
the role arn is `arn:aws:iam::760097843905:role/platform-developer` I think

Reply:
That worked! But there is also a `platform-dev` role in the IAM, which is maybe a bit confusing